fvm/src/call_manager/default.rs

```diff
@@ -309,18 +326,40 @@ where
             // From this point on, there are no more syscall errors, only aborts.
             let result: std::result::Result<RawBytes, Abort> = (|| {
                 // Lookup the invoke method.
-                let invoke: wasmtime::TypedFunc<(u32,), u32> = instance
+                let invoke: wasmtime::TypedFunc<(u32, ), u32> = instance
                     .get_typed_func(&mut store, "invoke")
                     // All actors will have an invoke method.
                     .map_err(Abort::Fatal)?;
 
                 // Invoke it.
-                let return_block_id = invoke.call(&mut store, (param_id,))?;
+                let res = invoke.call(&mut store, (param_id, ));
+                let fuel_consumed = match store
+                    .fuel_consumed() {
+                    Some(v) => v,
+                    None => 0,
+                };
```

Comment:
This will be the fuel consumed over the course of the entire execution (_including_ fuel manually consumed to account for syscalls). We need to charge for the fuel consumed since the last syscall.